offerzen/global/employers/employers@4.0.js

```diff
@@ -128,22 +128,27 @@ window.$loaded(function (window, document, $, undefined) {
   })
 
   const updateContactPhoneValue = function (event) {
-    dialCode.value = '+' + iti.getSelectedCountryData().dialCode
-    contact_phone.value =
-      dialCode.value +
-      input.value.replace(/[ \-\(\)]/g, '').replace(/\+([0-9]{2})|([0])/g, '')
+    dialCode.value = '+' + iti.getSelectedCountryData().dialCode;
+
+    const matchDialCode = new RegExp(`^(\\${dialCodeValue}|0)`); // only search start of string for matching code
```

Comment:
Where does dialCodeValue get defined? Sorry if I missed it, but had a look through the file and didn't see anything

Reply:
Ah, it's line 117 (`document.querySelector('#dial_code')`). I think what's confusing is I shadow the variable in the other function. Will rename there for clarity